pandas/core/indexes/base.py

```diff
@@ -2939,6 +2947,12 @@ def fillna(self, value=None, downcast=None):
         --------
         DataFrame.fillna : Fill NaN values of a DataFrame.
         Series.fillna : Fill NaN Values of a Series.
+
+        Examples
+        --------
+        >>> idx = pd.Index([None, np.nan, 3])
```

Comment:
although you're constructing with `None` and `np.nan`, they both get converted to `np.nan`
```
In [2]:  pd.Index([None, np.nan, 3])
Out[2]: Index([nan, nan, 3.0], dtype='float64')
```

To reduce the chance for confusion, shall we just construct with `np.nan` directly?

Reply:
Sure